tests/foreman/cli/test_setting.py

```diff
@@ -33,315 +34,352 @@
 
-        :id: 01ce95de-2994-42b6-b9f8-f7882981fb69
+@pytest.mark.stubbed
+@tier2
+def test_positive_update_email_delivery_method_sendmail():
+    """Check Updating Sendmail params through settings subcommand
 
-        :steps:
+    :id: 578de898-fde2-4957-b39a-9dd059f490bf
 
-            1. Execute "settings" command with "set" as sub-command
-            without any string(empty value) in value parameter
+    :steps:
+        1. set "delivery_method" to sendmail
+        2. set all sendmail related properties:
+            2.1. sendmail_arguments
+            2.2. sendmail_location
 
-        :expectedresults: Message on login screen should be removed
-        """
-        Settings.set({'name': "login_text", 'value': ""})
-        login_text = Settings.list({'search': 'name=login_text'})[0]
-        self.assertEqual('', login_text['value'])
+    :expectedresults: Sendmail properties are updated
 
-    @tier2
-    def test_positive_update_login_page_footer_text_with_long_string(self):
-        """Attempt to update parameter "Login_page_footer_text"
-            with long length string under General tab
+    :CaseImportance: Low
 
-        :id: 87ef6b19-fdc5-4541-aba8-e730f1a3caa7
+    :CaseAutomation: notautomated
+    """
 
-        :steps:
-            1. Execute "settings" command with "set" as sub-command
-            with long length string
 
-        :expectedresults: Parameter is updated
+@tier2
+@pytest.mark.parametrize('setting_update', ['email_reply_address'], indirect=True)
+def test_positive_update_email_reply_address(setting_update):
+    """Check email reply address is updated
 
-        :CaseImportance: Low
-        """
-        for login_text_value in generate_strings_list(1000):
-            with self.subTest(login_text_value):
-                Settings.set({'name': "login_text", 'value': login_text_value})
-                login_text = Settings.list({'search': 'name=login_text'})[0]
-                self.assertEqual(login_text_value, login_text['value'])
+    :id: cb0907d1-9cb6-45c4-b2bb-e2790ea55f16
 
-    @pytest.mark.stubbed
-    @tier2
-    def test_positive_update_email_delivery_method_smtp(self):
-        """Check Updating SMTP params through settings subcommand
+    :expectedresults: email_reply_address is updated
 
-        :id: b26798e9-528c-4ed6-a09b-dc8e4668a8d7
+    :CaseImportance: Low
 
-        :steps:
-            1. set "delivery_method" to smtp
-            2. set all smtp related properties:
-                2.1. smtp_address
-                2.2. smtp_authentication
-                2.3. smtp_domain
-                2.4. smtp_enable_starttls_auto
-                2.5. smtp_openssl_verify_mode
-                2.6. smtp_password
-                2.7. smtp_port
-                2.8. smtp_user_name
+    :parametrized: yes
 
-        :expectedresults: SMTP properties are updated
+    :CaseAutomation: automated
+    """
+    email_address = [email for email in valid_emails_list()][0]
```

Comment:
Same here

Reply:
valid_emails_list() return a list of random email.

```
[email for email in valid_emails_list()][0]
'arvqg6HNGP3Spc9PQsx6zpbKbEGTh82wlS3R46aJyP0D6WZx@example.com'
```

Reply:
Fixed